src/tcgwars/logic/impl/gen7/SunMoonPromos.groovy

```diff
@@ -2675,6 +2684,201 @@ public enum SunMoonPromos implements LogicCardInfo {
               flipThenApplySC PARALYZED
             }
           }
+        };
+            case RESHIRAM_CHARIZARD_GX_SM201:
+       return basic (this, hp:HP270, type:R, retreatCost:3) {
+         weakness W
+         move "Outrage", {
+           text "30+ damage. This attack does 10 more damage for each damage counter on this Pokémon."
+           energyCost R, C
+           onAttack {
+             damage 30+10*self.numberOfDamageCounters            
+          }
+         }
+         move "Flare Strike", {
+           text "230 damage. This Pokémon can't use Flare Strike during your next turn."
+           energyCost R, R, R, C
+           onAttack {
+             damage 230
+             cantUseAttack(thisMove, self)
+           }
+         }
+         move "Double Blaze GX", {
+           text "200+ damage. If this Pokémon has at least 3 extra [R] Energy attached to it (in addition to this attack's cost), this attack does 100 more damage, and this attack's damage isn't affected by any effects on your opponent's Active Pokémon. (You can't use more than 1 GX attack in a game.)"
+           energyCost R, R, R
+           attackRequirement {gxCheck()}
+           onAttack {
+             gxPerform()
+             if(self.cards.energySufficient(thisMove.energyCost + R + R + R)){
+               shredDamage 300
+             } else {
+               damage 200
+             }
+           }
+         }
+       };
+      case AMOONGUSS_SM202:
+        return evolution (this, from:"Foongus", hp:HP100, type:G, retreatCost:2) {
+          weakness R
+          bwAbility "Bursting Spores", {
+            text "Whenever you play a Pokémon that has the Spore attack from your hand during your turn, you may leave your opponent's Active Pokémon Asleep and Poisoned."
+            delayedA {
+              before PLAY_BASIC_POKEMON, {
+                if(ef.cardToPlay.moves.find{ it.name == "Spore" }) {
```

Comment:
Add this condition as well `&& bg.currentTurn == self.owner`, else it will work for both players.